Resources/translations/validators.en.yml

```diff
@@ -1 +1,2 @@
 craueFormFlow.previousStepInvalid: The form for step %stepNumber% is invalid. Please go back and try to submit it again.
+craueFormFlow.formExpired: This form has expired. Please resubmit your information.

```

Comment:
So how about `craueFormFlow.flowExpired: This form has expired. Please try to submit it again.`?